datadog/resource_datadog_role.go

```diff
@@ -90,32 +90,14 @@ func validatePermissionsUnrestricted(value interface{}, meta interface{}) error
 	return nil
 }
 
-func getRolePermissionSchema() *schema.Resource {
```

Comment:
This I'd keep here, it's for the schema it's interesting to have in the same file

Reply:
Yeah thats fair. The schema is used in tests, so it'd have to be exported, but I agree it should live here.
Thinking more, the Schema is pretty much the external API of a provider, so 👍 to exporting.